hutch_python/exp_load.py

```diff
@@ -0,0 +1,28 @@
+import logging
+from importlib import import_module
+
+from .utils import safe_load
+
+logger = logging.getLogger(__name__)
+
+
+def get_exp_objs(proposal, run):
```

Comment:
Enforcing a way to do this is a good idea. Standardization makes for easier maintainable code top to bottom